Dockerfile

```diff
@@ -67,9 +68,11 @@ RUN mkdir /usr/lib/jvm \
 
 # Copy client dist from previous docker build staget
 COPY --from=builder $CLIENT_DIST_DIR/* $SCORE_CLIENT_HOME/
+RUN groupadd -r score && useradd -r -g score score
```

Comment:
delete for reasons described in https://github.com/overture-stack/SONG/pull/558#discussion_r355610658